libraries/dawg-server-common/src/main/java/com/comcast/video/dawg/common/security/AuthServerConfig.java

```diff
@@ -0,0 +1,59 @@
+package com.comcast.video.dawg.common.security;
+
+public class AuthServerConfig {
+	private String ldapHost = "localhost";
+	private int ldapPort = 389;
+	private String ldapDomain = "dc=dawg,dc=comcast,dc=com";
```

Comment:
It does come from a config file, this is the default. Guess I could remove the default.